crates/stc_ts_type_checker/tests/conformance/types/mapped/recursiveMappedTypes.ts.stats.rust-debug

```diff
@@ -2,5 +2,5 @@ Stats {
     required_error: 9,
     matched_error: 0,
     extra_error: 0,
-    panic: 0,
+    panic: 1,
```

Comment:
It's Overflow of Recursive Values Issue

I think that We will should has solution about put dejavu in normalize
There may be other ways than this.

https://github.com/dudykr/stc/blob/36454c34724a476623376e771e9a7703697af328/crates/stc_ts_file_analyzer/src/analyzer/types/mod.rs#L526-L533